file-formats/cfdf/examples/yy1_aff_example.cfdf.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
@albertmink Does the check consider the schema added by this PR? Or do I miss an inconsistency in this PR.

Reply:
It does, see
<img width="933" alt="image" src="https://user-images.githubusercontent.com/38354196/193284449-00065e34-03b1-427a-9f41-beb227d76d90.png">